shared/constants/swaps.js

```diff
@@ -54,6 +55,14 @@ export const TEST_ETH_SWAPS_TOKEN_OBJECT = {
   iconUrl: TEST_ETH_TOKEN_IMAGE_URL,
 };
 
+export const RINKEBY_SWAPS_TOKEN_OBJECT = {
```

Comment:
I guess we should include this for QA purposes, but we should remove it before shipping to prod, because swaps does not actually work on rinkeby

Reply:
What if we just disable swapping for Rinkeby after we are done with QA, at least until our EIP 1559 changes are in production and working? It could be done by removing this line: 
https://github.com/MetaMask/metamask-extension/pull/11635/files#diff-9f9eff0a280f38efd9f071f24d503f51a1f7617a30f67e7e654e157dfc060902R98

Once we verify that EIP 1559 changes work well in production, I can send a quick PR for removing Rinkeby from Swaps.

Reply:
yeah, that makes sense